src/com/znd/smash/commands/SmashChamps.java

```diff
@@ -56,4 +56,124 @@ private static ItemStack createPyroMenuKit() {
+		ItemStack item = new ItemStack(Material.REDSTONE);
+		ItemMeta meta = item.getItemMeta();
+		meta.setDisplayName(ChatColor.DARK_RED + "Demon");
+		
+		List<String> lore = new ArrayList<String>();
+		lore.add("");
+		lore.add(ChatColor.RED + "Click to select kit!");
+		meta.setLore(lore);
+		
+		meta.addEnchant(Enchantment.DURABILITY, 1, true);
+		meta.addItemFlags(ItemFlag.HIDE_ENCHANTS);
+		item.setItemMeta(meta);
+		
+		return item;		
+	}
+	
+	private static ItemStack createKnightMenuKit() {
+		ItemStack item = new ItemStack(Material.IRON_SWORD);
+		ItemMeta meta = item.getItemMeta();
+		meta.setDisplayName(ChatColor.GRAY + "Knight");
+		
+		List<String> lore = new ArrayList<String>();
+		lore.add("");
+		lore.add(ChatColor.RED + "Click to select kit!");
+		meta.setLore(lore);
+		
+		meta.addEnchant(Enchantment.DURABILITY, 1, true);
+		meta.addItemFlags(ItemFlag.HIDE_ENCHANTS);
+		item.setItemMeta(meta);
+		
+		return item;		
+	}
+	
+	private static ItemStack createArcherMenuKit() {
+		ItemStack item = new ItemStack(Material.BOW);
+		ItemMeta meta = item.getItemMeta();
+		meta.setDisplayName(ChatColor.GREEN + "Archer");
+		
+		List<String> lore = new ArrayList<String>();
+		lore.add("");
+		lore.add(ChatColor.RED + "Click to select kit!");
+		meta.setLore(lore);
+		
+		meta.addEnchant(Enchantment.DURABILITY, 1, true);
+		meta.addItemFlags(ItemFlag.HIDE_ENCHANTS);
+		item.setItemMeta(meta);
+		
+		return item;		
+	}
+	
+	private static ItemStack createSorcererMenuKit() {
+		ItemStack item = new ItemStack(Material.BOOK);
+		ItemMeta meta = item.getItemMeta();
+		meta.setDisplayName(ChatColor.BLUE + "Sorcerer");
+		
+		List<String> lore = new ArrayList<String>();
+		lore.add("");
+		lore.add(ChatColor.RED + "Click to select kit!");
+		meta.setLore(lore);
+		
+		meta.addEnchant(Enchantment.DURABILITY, 1, true);
+		meta.addItemFlags(ItemFlag.HIDE_ENCHANTS);
+		item.setItemMeta(meta);
+		
+		return item;		
+	}
+	
+	private static ItemStack createNinjaMenuKit() {
+		ItemStack item = new ItemStack(Material.STRING);
+		ItemMeta meta = item.getItemMeta();
+		meta.setDisplayName(ChatColor.WHITE + "Ninja");
+		
+		List<String> lore = new ArrayList<String>();
+		lore.add("");
+		lore.add(ChatColor.RED + "Click to select kit!");
+		meta.setLore(lore);
+		
+		meta.addEnchant(Enchantment.DURABILITY, 1, true);
+		meta.addItemFlags(ItemFlag.HIDE_ENCHANTS);
+		item.setItemMeta(meta);
+		
+		return item;		
+	}
+	
+	private static ItemStack createIceMageMenuKit() {
+		ItemStack item = new ItemStack(Material.POWDER_SNOW_BUCKET);
+		ItemMeta meta = item.getItemMeta();
+		meta.setDisplayName(ChatColor.AQUA + "Ice Mage");
+		
+		List<String> lore = new ArrayList<String>();
+		lore.add("");
+		lore.add(ChatColor.RED + "Click to select kit!");
+		meta.setLore(lore);
+		
+		meta.addEnchant(Enchantment.DURABILITY, 1, true);
+		meta.addItemFlags(ItemFlag.HIDE_ENCHANTS);
+		item.setItemMeta(meta);
+		
+		return item;
+	}
+	
```

Comment:
Please add a screenshot of the GUI in the PR. Look up how to set up a spigot server and export the plugin as a jar then add it in the plugins folder.

Reply:
![Screenshot 2022-09-11 230637](https://user-images.githubusercontent.com/44095061/189584496-d671f549-09de-46f5-8fb6-7b935ea7c839.png)

Reply:
Good job!